build/xtask/src/main.rs

```diff
@@ -121,11 +120,28 @@ enum Xtask {
     },
 }
 
+#[derive(Clone, Debug, Parser)]
+pub struct HumilityArgs {
+    /// Path to the image configuration file, in TOML.
+    cfg: PathBuf,
+
+    /// Request verbosity from tools we shell out to.
+    #[clap(short, long)]
+    verbose: bool,
+
+    /// Extra options to pass to Humility
+    #[clap(last = true)]
+    extra_options: Vec<String>,
+}
+
 // For commands which may execute on specific packages, this enum
 // identifies the set of packages that should be operated upon.
```

Comment:
I know you didn't modify this line, but I now find it pretty confusing? it is supposedly describing an enum but is describing main?

Reply:
lol yeah, it looks like someone (probably me) forgot to delete it when removing `enum RequestedPackages`
```rust
// For commands which may execute on specific packages, this enum
// identifies the set of packages that should be operated upon.
enum RequestedPackages {
    // Specifies a single specific (Package, Target) pair.
    Specific(Option<String>, Option<String>),
    // Specifies the command should operate on all packages.
    All,
}
```

I deleted it now!